faqs/mkr-token.md

```diff
@@ -28,9 +28,9 @@ Another important responsibility for MKR holders is to communicate publicly abou
 
 ## What voting rights do MKR holders have?
 
-MKR holders have the ability to enact technical changes to the Maker Protocol itself and also to ratify decisions on-chain about non-technical matters. Whether that be a change to the system parameters, or a decision to accept a new governing philosophy, MKR voters have ultimate control. To learn more about governance in the Maker Protocol visit the [Governance FAQ](faqs/governance.md).
+MKR holders have the ability to enact technical changes to the Maker Protocol itself and also to ratify decisions on-chain about non-technical matters. Whether that be a change to the system parameters, or a decision to accept a new governing philosophy, MKR voters have ultimate control. To learn more about governance in the Maker Protocol visit the [Governance FAQ](scd-faqs/scd-faqs/governance.md).
 
-MKR holders who are interested in actively participating in the management of the Maker Protocol can visit the [Voter Onboarding Guide](insert link) for an overview of how to get started.
+MKR holders who are interested in actively participating in the management of the Maker Protocol can visit the [Voter Onboarding Guide](onboarding/voter-onboarding.md) for an overview of how to get started.
```

Comment:
> MKR holders who are interested in actively participating in the management of the Maker Protocol can visit the [Voter Onboarding Guide](onboarding/voter-onboarding.md) for an overview of how to get started.

this link still goes to a 404

Reply:
gah, ok. I think I know what it is.

I should be putting a 

./ before the onboarding/ so that it can reference properly one level deeper. 

right now it links to /faqs/onboarding/voter...

will push fix today!